### PR TITLE
feat: splashscreen

### PR DESCRIPTION
**Pull request by tomaszantas**
Description
---

- [x] Add splashscreen

The implementation does not follow the [Tauri docs](https://tauri.app/v1/guides/features/splashscreen/).
The difference is that the Tauri docs suggest to add separate `splashscreen.html` in distDir (`src/public/splashscreen.html`), and then toggle between splashscreen and main view (`src/public/index.html` React app) using the Tauri command. 

In this PR, the splashscreen is built as another div in the `index.html` that covers main application. Then, the React app hides the splashscreen by changing its `display` style to `none`. This implementation allows to apply fade out animation on the splashcreen.

Motivation and Context
---

#247 

How Has This Been Tested?
---

https://user-images.githubusercontent.com/11715931/178155406-edc09681-52ff-4807-a282-025a5b604352.mov

https://user-images.githubusercontent.com/11715931/178155443-d6f2b8c1-b684-4a98-b7b9-94a9b4bd5842.mov

### CODE REVIEW

File: applications/launchpad/gui-react/src/splashscreen.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+let start: number | null = null
+let opacity = 1
+
+const fadeOut = (timestamp: number, splashscreenEl: HTMLElement) => {
+  if (!start) start = timestamp
```
Comment (tarnas14) [inline applications/launchpad/gui-react/src/splashscreen.ts]:
```suggestion
  if (!start) {
    start = timestamp
  }
```
({ } make it more readable)

**Review by tarnas14 — CHANGES_REQUESTED** [review]

**Comment by tarnas14** [comment]
looks cool <3

File: applications/launchpad/gui-react/src/splashscreen.ts (hunk 1 above)
Comment (tomaszantas) [reply applications/launchpad/gui-react/src/splashscreen.ts]: Thanks! I will fix it soon

**Review by corquaid — APPROVED** [review]
Sweet.

**Review by tarnas14 — APPROVED** [review]

**Pull request merged**